silverback/runner.py

```diff
@@ -82,6 +83,21 @@ async def _event_task(
         handle an event handler task for the given contract event
         """
 
+    async def _cron_loop(self):
+        """
+        Execute scheduled cron tasks when their sequence comes up.
+        """
+        while True:
+            await asyncio.sleep(CRON_TICK)
```

Comment:
what happens if the cronjob is scheduled to be less than 15 seconds apart?

Reply:
Cron has a resolution of 1 minute.

Reply:
I reduced his to 5 just to be a little more accurate

Can do like a drift sort of logic to make it loosely align with :00 +/- 1 sec (as clock drift may occur)